query-cloudtrail.sh

```diff
@@ -24,7 +35,7 @@ for bucket in $OLD_BUCKETS; do
   set -x
   aws cloudtrail lookup-events \
     --lookup-attributes AttributeKey=ResourceName,AttributeValue="$bucket" \
-    | jq -c "$JQ_TRANSFORM"
+    | jq "$JQ_QUERY"
```

Comment:
I would recommend `jq -r` if you plan on doing anything with the output so the output is rendered as raw JSON. Not a requirement, just a suggestion.

Reply:
I don't think that would work, here.  `jq -r` is for raw strings, not raw json.  ie: